.travis.yml

```diff
@@ -3,8 +3,14 @@ services: docker
 script: docker build . -t cs50/cli
 before_deploy: docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 deploy:
-  provider: script
+- provider: script
   script: docker push cs50/cli
+  on:
+    branch: master
```

Comment:
Should be `ubuntu` branch!

* `cs50/cli` and `cs50/cli:latest` should be based on `master` branch.
* `cs50/cli:ubuntu` should be based on `ubuntu` branch.

Reply:
There are two deployment providers, this one is for master (pushes cs50/cli:latest), and https://github.com/cs50/cli/pull/20/files#diff-354f30a63fb0907d4ad57269548329e3R13 is for ubuntu (pushes cs50/cli:ubuntu).

Reply:
Oh  phew, I see, thanks.